osbs/http.py

```diff
@@ -212,4 +212,10 @@ def json(self, check=True):
         if check and self.status_code not in (0, requests.codes.OK, requests.codes.CREATED):
             raise OsbsResponseException(text, self.status_code)
 
-        return json.loads(text)
+        try:
+            return json.loads(text)
+        except ValueError:
+            msg = '{0}Headers {1}\nContent {2}'.format('HtttpResponse has corrupt json:\n',
```

Comment:
Seems odd to have the literal string being formatted to msg. Can this be changed to:
```python
msg = 'HtttpResponse has corrupt json:\nHeaders {0}\nContent {1}'.format(self.headers, self.content)
```

Reply:
It could be, but then the line break is at column 92 and self.headers, self.content are formatted ugly.